packages/autorag/bff/docs/maas-models-endpoint.md

```
@@ -0,0 +1,216 @@
+cd packages/autorag/bff
+make run MOCK_K8S_CLIENT=true MOCK_MAAS_CLIENT=true
+```
+
+```bash
+curl 'http://localhost:4000/api/v1/maas/models?namespace=default&secretName=any-secret'
+```
+
+### Full E2E
+
+1. Create a secret with Models as a Service credentials. Use an in-cluster service DNS name (loopback URLs such as `http://localhost:8321` are rejected):
+   ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)